### ISSUE TITLE
Bronze is too high level

### ISSUE THREAD

**Issue by momdo** [w3c/silver#284]
@MakotoUeki -san says that welcome feedback in Japanese, so I write it in Japanese for accuracy.

Bronzeは3.4 Conformance LevelsのEditor's noteで、

> We expect that bronze will be similar to WCAG 2 AA, while silver and gold will include more usability-type testing.

とされていますが、現在のWCAG 3.0 FPWDでは、Bronzeは最低レベルとして定義されています。つまり、WCAG 2.x level Aよりも高いレベルがWCAG 3.0での最低のレベルということになります。

最低レベルを引き上げることが、世界中のウェブコンテンツをよりアクセシブルにすることにつながるのでしょうか？確かにレベルを引き上げることは、既存のウェブアクセシビリティのガイドラインに取り組めているサイトが、よりアクセシビリティを高めることにつながるでしょう。その一方で、残念ながらまだウェブアクセシビリティに取り組めていないサイトに対して、ウェブアクセシビリティに取り組むことをより困難にすることにつながると考えます。取り組むことが困難であるならば、最初からウェブアクセシビリティに取り組まなくてもよいという諦めにつながるかもしれません（たとえ、WCAG 2.xとWCAG3とが当面並立するにしても、WCAG3勧告となったときに、WCAG3が既にあるにもかかわらず難解なWCAG 2.xを用いて取り組みたいという人は多くはないでしょう）。

WCAG3 BronzeをWCAG 2.x level AA相当にするならば、ウェブアクセシビリティにより取り組みやすくするためにも、WCAG 2.x level A相当の、ある基準を満たせば最低限のウェブアクセシビリティを確保できているとみなすことができる適合レベルを提供すべきと考えます。

それとは別に、たとえWCAG3のなんらか適合レベルを満たせていなくても、WCAG3はスコアリングで定量的に示すことができる仕組みになっています。スコアリングを用いることで、ある時点でここまで出来ているということを宣言できる方法も取ることができると考えられます。これによりサステナビリティを確保できる、言い換えると継続的にウェブアクセシビリティを高めていく取り組みにつなげることができると思います。このような仕組みについても検討してみてはいかがでしょうか。

**Comment by jspellman** [comment]
From Google Translate: However, in the current WCAG 3.0 FPWD, Bronze is defined as the lowest level. This means that a level higher than WCAG 2.x level A is the lowest level in WCAG 3.0.

Will raising the minimum level make web content around the world more accessible? Certainly, raising the level will lead to sites that are working on existing web accessibility guidelines to be more accessible. On the other hand, unfortunately, it will make it more difficult to work on web accessibility for sites that have not yet worked on web accessibility. If it's difficult to tackle, it may lead to giving up on not having to tackle web accessibility from the beginning (even if WCAG 2.x and WCAG3 are side by side for the time being, it's a WCAG3 recommendation. Sometimes, not many people want to work with the esoteric WCAG 2.x, even though they already have WCAG3).

If WCAG3 Bronze is equivalent to WCAG 2.x level AA, in order to make it easier to work on web accessibility, it is said that the minimum web accessibility equivalent to WCAG 2.x level A can be ensured if certain criteria are met. We believe that we should provide a level of conformance that can be considered.

Apart from that, WCAG3 is a mechanism that can be quantitatively shown by scoring even if it does not meet any conformance level of WCAG3. By using scoring, it is possible to take a method that can declare that it is done so far at a certain point. I think this will lead to efforts to ensure sustainability, in other words, to continuously improve web accessibility. How about considering such a mechanism as well?

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
